grails-app/assets/javascripts/charts2.js

```diff
@@ -378,19 +480,21 @@ var taxonomyChart = {
         var dataTable = new google.visualization.DataTable();
         dataTable.addColumn('string', chartLabels[name] ? chartLabels[name] : name);
         dataTable.addColumn('number','records');
-        $.each(data.taxa, function(i,obj) {
+        $.each(data.taxa, function(i, obj) {
             dataTable.addRow([obj.label, obj.count]);
         });
 
         // resolve the chart options
         var opts = $.extend({}, taxonomyPieChartOptions);
         opts = $.extend(true, opts, this.chartOptions);
-        opts.title = opts.name ? opts.name + " records by " + data.rank : "By " + data.rank;
-        opts.backgroundColor = { fill:'transparent' };
+        opts.title = opts.name ? opts.name + ' records by ' + data.rank : 'By ' + data.rank;
+        opts.backgroundColor = {
+            fill: 'transparent'
+        };
 
         // create the outer div that will contain the chart and the additional links
         var $outerContainer = $('#taxa');
-        if ($outerContainer.length == 0) {
+        if(!$outerContainer.length) {
```

Comment:
Same here. Basically, boolean check is for ``false/undefined/null`` and if you're feeling lucky, strings as well.